Code/Framework/AzCore/Tests/AZStd/Hashed.cpp

```diff
@@ -1420,7 +1420,11 @@ namespace UnitTest
     TYPED_TEST_CASE(HashedSetDifferentAllocatorFixture, SetTemplateConfigs);
 
 #if GTEST_HAS_DEATH_TEST
+#if AZ_TRAIT_DISABLE_FAILED_DEATH_TESTS
+    TYPED_TEST(HashedSetDifferentAllocatorFixture, DISABLED_InsertNodeHandleWithDifferentAllocatorsLogsTraceMessages)
```

Comment:
Do you know what the issue with the Google Death test are?
Is it not correctly causing a non-zero exit code?

Reply:
I'll have to run them again, but encountering one would prevent running the remaining tests.

Reply:
Ok then it is probably an issue with the Android runtime then, where it maybe having issues forking processes in an isolated manner..

Reply:
It does log a warning about forking process with threads, but the test itself is aborting while operating on a null `FILE*`.  Needs some investigation.